### DISCUSSION TITLE
How to run more than 1 test in same context, Adding before all is not working.

### DISCUSSION THREAD

**Discussion by mrigankErisk** [akshayp7/playwright-typescript-playwright-test#30] [Q&A]
@akshayp7  How to run more than 1 test in same context, Adding before all is not working.Can you please let me know how to do that.
or If you can add support for before all in test

got this error.

![image](https://user-images.githubusercontent.com/99946156/174842230-248a1064-92b5-44bd-9eb1-f61754c872a2.png)

**Comment by akshayp7** [comment]
Currently playwright advocates that single tets case must be run in a new clean browser so that previous sessions data doesn't affect the new test case. I am looking into this issue will get back if I find something.

**Reply by mrigankErisk** [reply]
Thnks
